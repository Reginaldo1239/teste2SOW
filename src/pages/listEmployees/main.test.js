import React from 'react';
import renderer from 'react-test-renderer';

import Main from './main';


test('renders a message', () => {
    const tree = renderer.create(<Main></Main>)
    .toJSON();
  expect(tree).toMatchSnapshot();
 
  })  