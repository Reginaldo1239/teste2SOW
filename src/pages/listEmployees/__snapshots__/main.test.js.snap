// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders a message 1`] = `
<div
  className="container"
>
  <div
    className="container"
  >
    <div
      className="ui icon input input"
    >
      <input
        onChange={[Function]}
        placeholder="buscar"
        type="text"
      />
      <i
        aria-hidden="true"
        className="search circular link icon"
        onClick={[Function]}
      />
    </div>
  </div>
  <div
    className="list"
  >
    <div
      className="ui inverted dimmer"
      onClick={[Function]}
    >
      <div
        className="content"
      >
        <div
          className="ui text loader"
        >
          Loading
        </div>
      </div>
    </div>
  </div>
  <div
    className="pagination"
  />
</div>
`;
